src/types.rs

```diff
@@ -399,16 +380,19 @@ impl EventData {
     where
         S: AsRef<str>,
     {
+        let mut metadata = HashMap::new();
+        metadata.insert("type".to_owned(), event_type.as_ref().to_owned());
+        metadata.insert("content-type".to_owned(), "application/json".to_owned());
```

Comment:
The content-type should be `application/octet-stream` in this case.

Reply:
oooh! good catch!  Fixed